docs/src/main/sphinx/functions/datetime.md

```diff
@@ -570,5 +570,14 @@ Returns the year of the [ISO week] from `x`.
 This is an alias for {func}`year_of_week`.
 :::
 
+:::{function} timezone(timestamp(p) with time zone) -> varchar 
```

Comment:
Please verify that the output HTML actually shows all the content .. we had issues with that in the past

Reply:
I confirmed .. its renders fine.